CorePush/CorePush.csproj

```diff
@@ -39,6 +39,8 @@ Previous Versions:
 
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="System.Security.Cryptography.Cng" Version="5.0.0" />
```

Comment:
Can this be deleted?

Reply:
Probably no but please confirm.

Reply:
Yeah, it works :-)

Reply:
@andrei-m-code can you merge and publish a version?

Also there are a few other small PRs there.